.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ jobs:
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: Build
-        run: zig build
+        run: zig build -Doptimize=ReleaseFast
```

Comment:
We should build and test in different release modes but we should use a matrix to do it so we test every release mode on every platform